oss.krtirtho.Spotube.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-desktop-file: spotube.desktop
+rename-appdata-file: spotube.appdata.xml
+command: spotube
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=~/.local/share/oss.krtirtho.spotube:create
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this?

Reply:
I'm not sure. I saw that in a blogpost maybe. Sorry for being too lazy to take no care for that part
I tested without it  & it seems to work fine. So I'm removing that line